deploy/ubuntu-config/proxy.ini

```diff
@@ -53,6 +53,11 @@ bind-service-account = false
 # If `bind-service-account` is set to false, but `allow-search` is set to true, only the DNs
 # in `passthrough-binds` will be able to issue search requests.
 allow-search = false
+# Currently, ldaptor does not support result references which will break
+# some apps in combination with AD. This option provides a workaround:
+# If it is set to `true`, the LDAP proxy ignores all LDAP SEARCH result references.
+# However, this means that the app does not receive any LDAP SEARCH references.
```

Comment:
To me this senstence is confusing. Why not:

If it is set to `true`, the LDAP proxy ignores all LDAP SEARCH result references. The LDAP Proxy does not send any LDAP SEARCH result references to the application.